shared/properties-view/properties-view.cpp

```diff
@@ -25,21 +25,30 @@
 #include <QObject>
 #include <QDesktopServices>
 #include "double-slider.hpp"
-#include "slider-ignorewheel.hpp"
 #include "spinbox-ignorewheel.hpp"
-#include "qt-wrappers.hpp"
 #include "properties-view.hpp"
 #include "properties-view.moc.hpp"
-#include "plain-text-edit.hpp"
-#include "obs-app.hpp"
 
+#include <qt-wrappers.hpp>
+#include <plain-text-edit.hpp>
+#include <slider-ignorewheel.hpp>
 #include <cstdlib>
 #include <initializer_list>
 #include <obs-data.h>
 #include <obs.h>
 #include <qtimer.h>
 #include <string>
 
+#ifdef INSIDE_UI
```

Comment:
Wouldn't it make more sense to include the frontend-API by default and use `obs_frontend_is_theme_dark` instead of making the distinction?

Reply:
In my opinion, UI (`obs-studio`) must not depend on obs-frontend-api since it's implementation/interface is provided by the former.

Reply:
I don't think that using an API internally is an issue - it would be a "bad" API if that weren't viable and this is precisely the use case for such an API that abstracts the implementation detail of the state variable.

IMO it's desirable to use the API instead of accessing global objects directly (that the implementation should not even know about in the first place).

Reply:
Since I want this to get merged, I'll go with removing `INSIDE_UI` and use only the API.